public/stylesheets/masters/low_vision_default/low_vision_default_site_screen_.css

```diff
@@ -185,3 +185,12 @@ pre {
 .bookmark .recent .index {
   border: none;
 }
+
+html.no-js form#work-filters {
+  display: block !important;
+}
+
+html.no-js p.narrow-hidden,
+html.no-js p.narrow-shown {
+  display: block !important;
+}
```

Comment:
This doesn't seem to do what the issue is looking for. The "Filters" button and "Top of Work Index" link should always be present in Low Vision Default, regardless of what size screen you're on or whether you have JavaScript enabled. I think you can do that by just copying the following CSS from the narrow stylesheet:

```css
body .narrow-shown {
  display: block;
}

.actions li.narrow-shown {
  display: inline;
}
```